guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -68,39 +74,70 @@ If you lose connection to the command shell where the upgrade command is running
 ----
 # {foreman-maintain} upgrade run --target-version {TargetVersionMaintainUpgrade}
 ----
-
+endif::[]
+ifdef::katello[]
+. Check for running tasks
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake katello:upgrade_check
+----
+. Update operating system packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y
+----
+. Update repositories
++
+.For Centos 7 or Red Hat Enterprise Linux Users:
+[options="nowrap" subs="attributes"]
+----
+# yum update -y https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+# yum install -y centos-release-scl-rh
+----
+. Update packages
+Clean the yum cache and update required packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum clean all
+# yum -y update
+----
+. Run installer
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-installer}
+----
+endif::[]
 . Check when the kernel packages were last updated:
 +
 [options="nowrap"]
 ----
 # rpm -qa --last | grep kernel
 ----
-
 . Optional: If a kernel update occurred since the last reboot, stop the `{foreman-maintain}` services and reboot the system:
 +
 ----
 # {foreman-maintain} service stop
 # reboot
 ----
-
 . If using a BASH shell, after a successful or failed upgrade, enter:
 +
 [options="nowrap"]
 ----
 # hash -d {foreman-maintain} service 2> /dev/null
 ----
-
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
```

Comment:
IMHO this is unsupported and we should not document it. https://theforeman.org/plugins/katello/nightly/installation/index.html#multiple-subnets-and-domains documents how to handle this in the installer. Perhaps out of scope for this PR.

Reply:
While i agree, i think maybe out of scope for this pr, maybe we should open an issue to incorporate https://theforeman.org/plugins/katello/nightly/installation/index.html#multiple-subnets-and-domains